### PR TITLE
Added Views

### PR DESCRIPTION
**Pull request by BryteLitty**
In this PR, I added view files

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,26 @@
...
+        <%= render partial: '/shared/user_photo', locals: { user: @user } %>
+        <%= render partial: '/shared/user_details', locals: { user: @user } %>
+    </div>
+
+    <div class="bio">
+        <h3>Bio</h3>
+        <p> <%= @user.bio %> </p>
+    </div>
+
+    <% @three_recent_posts.each do |post| %>
```
Comment (victorokoroji) [inline app/views/users/show.html.erb]: - [ ] Good job so far 👍🏽 , I could see that you use the method @three_recent_posts but in your controller it says @recent_posts. KIndly correct that 👍🏽

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1 +1,20 @@
...
+        <span>Likes: <%= @post.likes_counter %></span>
+    </div>
+    <p><%= @post.text %></p>
+    <h3>Comments:</h3>
+    
+    <% @post.comments.each do |comment| %>
+    <div class="post-comments">
+        <span><%= comment.author.name %>:</span>
+        <span><%= comment.text %></span>
+    </div>
```
Comment (victorokoroji) [inline app/views/posts/show.html.erb]: - [ ] Good job so far 👍🏽 . Kindly use the  @recent_comments to render 5 most recent comments as specified in the project requirement and remember tp add the method to the show of your posts controller 👍🏽

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi  @BryteLitty, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- Controllers well written✅ 
- Good understanding of how views work ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @BryteLitty 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**